grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx

```diff
@@ -1,241 +1,82 @@
-                        </Tooltip>
-                      </HorizontalGroup>
-                    )}
-                    <HorizontalGroup>
-                      <PluginLink target="_blank" query={{ page: 'integrations', id: selectedTeamDirectPaging.id }}>
-                        <IconButton
-                          tooltip="Open integration in new tab"
-                          style={{ color: 'var(--always-gray)' }}
-                          name="external-link-alt"
-                        />
-                      </PluginLink>
-                    </HorizontalGroup>
-                  </HorizontalGroup>
-                </li>
-              </ul>
-              {!escalationChainsExist && (
-                <Alert severity="warning" title="Direct paging integration not configured">
-                  <VerticalGroup>
-                    <Text>
-                      The direct paging integration for the selected team has no escalation chains configured.
-                      <br />
-                      If you proceed with the alert group, the team likely will not be notified. <br />
-                      <a
-                        href={
-                          'https://grafana.com/docs/oncall/latest/integrations/manual/#learn-the-flow-and-handle-warnings'
-                        }
-                        target="_blank"
-                        rel="noreferrer"
-                        className={cx('link')}
-                      >
-                        <Text type="link">Learn more.</Text>
-                      </a>
-                    </Text>
-                  </VerticalGroup>
-                </Alert>
-              )}
-            </VerticalGroup>
-          ) : (
-            <Alert severity="warning" title={'Direct paging integration missing'}>
-              <HorizontalGroup>
-                <Text>
-                  The selected team doesn't have a direct paging integration configured and will not be notified. <br />
-                  If you proceed with the alert group, an empty direct paging integration will be created automatically
-                  for the team. <br />
-                  <a
-                    href={
-                      'https://grafana.com/docs/oncall/latest/integrations/manual/#learn-the-flow-and-handle-warnings'
-                    }
-                    target="_blank"
-                    rel="noreferrer"
-                    className={cx('link')}
-                  >
-                    <Text type="link">Learn more.</Text>
-                  </a>
-                </Text>
-              </HorizontalGroup>
-            </Alert>
-          ))}
-      </VerticalGroup>
-    );
-  };
+  const [formIsValid, setFormIsValid] = useState<boolean>(false);
+
+  const onHideDrawer = useCallback(() => {
+    directPagingStore.resetSelectedUsers();
+    directPagingStore.resetSelectedTeam();
+    onHide();
+  }, [directPagingStore, onHide]);
```

Comment:
`directPagingStore` should never change, so most likely you don't need to mark it as a dependency